tina/util/transformations.py

```diff
@@ -0,0 +1,1705 @@
+# -*- coding: utf-8 -*-
```

Comment:
Why not just install https://pypi.org/project/transformations/ ?

Reply:
Sure. I update the code!